pkg/discovery/processor/cluster_processor.go

```diff
@@ -229,6 +229,14 @@ func (p *ClusterProcessor) DiscoverCluster() (*repository.ClusterSummary, error)
 	// Discover Turbo Policies
 	NewTurboPolicyProcessor(p.clusterInfoScraper, kubeCluster).ProcessTurboPolicies()
 
+	// Discover GitOps configuration overrides
+	NewGitOpsConfigProcessor(p.clusterInfoScraper, kubeCluster).ProcessGitOpsConfigs()
+
+	// Update the GitOps configuration override cache
+	if clusterScraper, ok := p.clusterInfoScraper.(*cluster.ClusterScraper); ok {
```

Comment:
Why do we need the type check here?  We already defined and implemented the `GetAllGitOpsConfigurations()` interface.

Reply:
Removed this code.